src/lint/collect-spelling-diagnostics.ts

```diff
@@ -65,6 +65,10 @@ let matchers = [
     message:
       'clauses should be referenced using <emu-xref> tags rather than hardcoding clause numbers',
   },
+  {
+    pattern: /(?<=[^ \n]) +<\/p>/gu,
+    message: '<p> tags should not contain trailing whitespace',
```

Comment:
should any tags contain trailing whitespace?

Reply:
`<td>` and `<th>` can have trailing whitespace for aligning cells in the source. Can't think of any others off the top of my head.

Reply:
can have, but should they? html compresses all consecutive whitespace down to a single space, so any spaces over one would be redundant, no?

Reply:
It's for alignment _in the source_. Specifically, we have some tables in 262 which are written like

```html
<figure>
  <table class="lightweight-table">
    <tbody>
      <tr><th> _assignmentOpText_ </th><th> _opText_       </th></tr>
      <tr><td> `**=`              </td><td> `**`           </td></tr>
      <tr><td> `*=`               </td><td> `*`            </td></tr>
      <tr><td> `/=`               </td><td> `/`            </td></tr>
      <tr><td> `%=`               </td><td> `%`            </td></tr>
      <tr><td> `+=`               </td><td> `+`            </td></tr>
      <tr><td> `-=`               </td><td> `-`            </td></tr>
      <tr><td> `&lt;&lt;=`        </td><td> `&lt;&lt;`     </td></tr>
      <tr><td> `&gt;&gt;=`        </td><td> `&gt;&gt;`     </td></tr>
      <tr><td> `&gt;&gt;&gt;=`    </td><td> `&gt;&gt;&gt;` </td></tr>
      <tr><td> `&amp;=`           </td><td> `&amp;`        </td></tr>
      <tr><td> `^=`               </td><td> `^`            </td></tr>
      <tr><td> `|=`               </td><td> `|`            </td></tr>
    </tbody>
  </table>
</figure>
```
which I don't want the linter to forbid.

Reply:
ah, ok. that seems pretty gross to me in the source, but i'll ack that you're not trying to block it now :-)